src/Core/src/Core/ISafeAreaView3.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Provides functionality for per-edge safe area control.
+	/// </summary>
+	/// <remarks>
+	/// This interface extends ISafeAreaView to provide fine-grained control over safe area behavior per edge.
+	/// This interface is primarily recognized on the iOS/Mac Catalyst platforms; other platforms may have limited support.
+	/// </remarks>
+	public interface ISafeAreaView3 : ISafeAreaView
```

Comment:
Can you just add this to ISafeAreaView2? And just implement this explicitly so we don't make it public yet